src/Facades/Qr.php

```diff
@@ -210,6 +241,8 @@ public static function getFormSchema(
     public static function output(?string $data = null, ?array $options = null): HtmlString
     {
         $maker = new Generator;
+        // $maker->errorCorrection('L');
```

Comment:
if we added the `percentage` to the default array we dont need this default variable